.github/workflows/update_version.yml

```diff
@@ -58,3 +58,35 @@ jobs:
                   release_name: ${{ steps.tag.outputs.new_tag }}
                   commitish: master
                   body: "Upgrade PHPStan to ${{ steps.tag.outputs.new_tag }}"
+
+    publish_docker_images:
+        needs: [update-version]
+        runs-on: ubuntu-20.04
+       
+        if: github.ref == 'refs/heads/master' || github.event_name == 'release'
+        steps:
+        - name: Checkout
+          uses: actions/checkout@v2
```

Comment:
v3 ?

Reply:
I haven't already tried v3

Reply:
Should work without issues, but lets keep it for now